src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 4;
+    private static final int BOUND_RANDOM = 100;
+    private static final int CIRCLE_RADIUS = 10;
+    private Random randomNumber = new Random();
+    private Random randomFigure = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int indexFigure = randomFigure.nextInt(FIGURES_COUNT);
+        if (indexFigure == 0) {
+            return new Square(randomNumber.nextInt(BOUND_RANDOM), colorSupplier.getRandomColor());
```

Comment:
```suggestion
            return getRandomSquare();
```
lets add private methods to get each figure - this would make your code easier to read